blast_config/tx_struct_tng.c

```diff
@@ -325,6 +325,46 @@ channel_t channel_list[] =
     /*                                      Other subsystem                                  */
     /*---------------------------------------------------------------------------------------*/
 
+    // Power LabJacks
+
+    // Outer Frame PBOB
+    { "current_fc1",      SCALE(CRYO_D), TYPE_UINT16, RATE_1HZ, U_V_V, 0 },
```

Comment:
What is CRYO_D? I could not find it defined anywhere.
Why is CRYO_D? Some of these things are not cryo-related.